src/components/expression/expression.tsx

```diff
@@ -123,18 +124,30 @@ export const EuiExpression: React.FunctionComponent<
       'euiExpression-isUppercase': uppercase,
     },
     displayToClassNameMap[display],
-    colorClass
+    colorToClassNameMap[calculatedColor]
   );
 
   const Component = onClick ? 'button' : 'span';
 
+  const descriptionStyle = descriptionProps && descriptionProps.style;
   const customWidth =
     display === 'columns' && descriptionWidth
       ? {
-          flexBasis: `${descriptionWidth}%`,
+          flexBasis: descriptionWidth,
+          ...descriptionStyle,
         }
       : undefined;
 
+  const invalidIcon = isInvalid ? (
+    <EuiIcon
+      className="euiExpression__icon"
```

Comment:
It's usually easier to just pass a class directly to the icon instead of creating a wrapper.